lib/liquid/tags/render.rb

```diff
@@ -31,7 +31,7 @@ class Render < Tag
 
     disable_tags "include"
 
-    attr_reader :template_name_expr, :variable_name_expr, :attributes
+    attr_reader :template_name_expr, :variable_name_expr, :attributes, :alias_name, :is_for_loop
```

Comment:
I think a predicate method would still be more idiomatic.  We shouldn't let instance variable name restrictions affect the desired API

```suggestion
    attr_reader :template_name_expr, :variable_name_expr, :attributes, :alias_name
    
    def for_loop?
      @is_for_loop
    end
```